GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -277,6 +277,7 @@ public FormBrowse(GitUICommands commands, BrowseArguments args)
                 PluginRegistry.Initialize();
                 await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
                 RegisterPlugins();
+                fileTree.RegisterGitHostingPluginInBlameControl();
```

Comment:
pushed a fix (trival, same as for fileTree) in my branch fix_lost_feature_github_from_blame
+2 with this

Reply:
@gerhardol Thanks for having done that because I had not the good value in the settings and was not able to see how to reproduce. Testing done and it is working good.
I think we can merge....